### PR TITLE
Add launch file package

### PR DESCRIPTION
**Pull request by mfehr**
... as a first step to declutter maplab_node, rovioli and maplab_server launch file folders.
This PR looks like a lot, but is merely moving the lidarstick launch files and calibrations and such over as an example and refactored what was needed for project Alice, the remaining launch files will need to be ported by whoever needs them.  There is a short description how the launch file package is supposed to be structured, let me know if you see any issues with this.

### CODE REVIEW

File: applications/maplab-server-node/src/maplab-server-node.cc [hunk 1]
```diff
@@ -527,7 +527,7 @@ bool MaplabServerNode::loadAndProcessSubmap(
 
 bool MaplabServerNode::saveMap() {
   std::lock_guard<std::mutex> lock(mutex_);
-  if (!FLAGS_maplab_server_merged_map_folder.empty()) {
```
Comment (floriantschopp) [inline applications/maplab-server-node/src/maplab-server-node.cc]: good catch

File: maplab-launch/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+## maplab_launch
+
+This package contains the robot/sensor specific launch files for:
```
Comment (floriantschopp) [inline maplab-launch/README.md]: launch files and calibrations, right?

File: maplab-launch/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ - maplab_node
+ - rovioli
+ - maplab_server_node
+
+The intended folder layout is as follows:
+
+```bash
+The intended folder structure is as follows:
```
Comment (floriantschopp) [inline maplab-launch/README.md]: duplication of line 8

File: maplab-launch/launch/lidarstick/lidarstick-maplab-node-w-okvis.launch [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+	<!-- MAPLAB -->
+	<node name="maplab_node" pkg="maplab_node" type="maplab_node" output="screen" clear_params="true">
+
+		<!-- GENERAL PARAMS -->
+		<rosparam command="load" file="$(find maplab_launch)/config/lidarstick/ros/lidarstick-maplab-node-rosparam.yaml"/>
+
+		<!-- ROBOT SPECIFIC PARAMS -->
+		<!-- SENSORS -->
+		<param name="sensor_calibration_file" value="$(find maplab_launch)/config/lidarstick/calib/lidarstick-150-deg-cams-sensors.yaml"/>
+		<param name="imu_to_camera_time_offset_ns" value="19779092"/>
```
Comment (floriantschopp) [inline maplab-launch/launch/lidarstick/lidarstick-maplab-node-w-okvis.launch]: are you supposed to put configuration here or in the rosparam file?

File: maplab-launch/README.md (hunk 2 above)
Comment (mfehr) [reply maplab-launch/README.md]: yes

File: maplab-launch/launch/lidarstick/lidarstick-maplab-node-w-okvis.launch (hunk 4 above)
Comment (mfehr) [reply maplab-launch/launch/lidarstick/lidarstick-maplab-node-w-okvis.launch]: the idea was to use a few rosparam files that cover the different use-cases/projects and the set/overwrite the robot specific settings and other settings that you change often (e.g. whether or not resources should be attached) in the launch file. I think this makes it very compact and is a good trade-off between having a stable launch file for each robot and not having all these large copy-pasted launch files or param files. Wdyt?

**Comment by mfehr** [comment]
@floriantschopp I split the voxblox dependency change into this PR: https://github.com/ethz-asl/maplab/pull/206

File: maplab-launch/launch/lidarstick/lidarstick-maplab-node-w-okvis.launch (hunk 4 above)
Comment (floriantschopp) [reply maplab-launch/launch/lidarstick/lidarstick-maplab-node-w-okvis.launch]: Agreed, makes sence. Maybe we should put that strategy somewhere...

**Pull request merged**